.github/workflows/ci.yml

```diff
@@ -104,6 +104,12 @@ jobs:
         rm -rf "$HOME/.sbt/scripted/" || true
         rm -rf "$HOME/.ivy2/local" || true
         rm -r $(find $HOME/.sbt/boot -name "*-SNAPSHOT") || true
+    - name: Publish local Zinc
+      shell: bash
+      run: |
+        ./sbt -v lowerUtils/publishLocal
+        cd zinc/
+        sbt publishLocal
```

Comment:
This branch depends on a SNAPSHOT version of Zinc. This will be the case until we publish the utils and Zinc.